src/Compatibility/Directory.Packages.props

```diff
@@ -8,8 +8,4 @@
     <PackageVersion Update="Microsoft.CodeAnalysis.CSharp" Version="4.4.0" />
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageVersion Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="$(MicrosoftCodeAnalysisWorkspacesCommonPackageVersion)" />
```

Comment:
@ericstj this was not needed anymore. It's in the Versions.props. I forgot to remove it in this PR, it was a temporary workaround while I merged the PR that introduced the new Versions.props and Version.Details.xml entries, which have been merged.